rakefile.rb

```diff
@@ -95,6 +101,9 @@
 desc "Execute default tasks"
 task :default => [ :vars, :gitlink, :unit, :integ, :spec, :pack ]
 
+desc "Execute default tasks plus .NET Core tasks"
+task :dotnet => [ :default, :unit_dotnet, :pack_dotnet]

```

Comment:
Indeed. I think the `dotnet` task should be what `default` would be if we were taking a dep on the half baked tooling.